2019Labs/CustomSecurityContent/documentation/lab2_profiles.adoc

```diff
@@ -0,0 +1,403 @@
+** Each product has a different set of profiles. The profiles represent a specific policy or benchmark, eg. “Protection profile for general purpose operating systems”.
+** The profiles are located in `profiles` directory in the product directory. The profiles are represented by a simple YAML (YAML Ain't Markup Language) file, eg. `ospp.profile` defines a profile.
+** Notice that there are already some `.profile` files in the `profiles` directory. You can get inspired by them.
+. Create new file `travel.profile` there and open it in the editor.
++
+----
+gedit travel.profile
+----
++
+. Create the basic structure and fill in the profile title and description as specified in this listing.
++
+----
+documentation_complete: true
+
+title: Travel profile for corporate laptops
+
+description: This profile represents settings which are required by company security policy for employee laptops.
+
+selections:
+    - security_patches_up_to_date
+----
++
+** Profile is a file in YAML format.
+** You should be fine if you copy and paste the content from the previous listing. If you attempt to create a YAML file by hand, the most common mistake tends to be using incorrect indentation. Make sure you use spaces, not tabs. Also check if no whitespace is trailing.
+** The profile consists of 4 items which are required:
+... `documentation_complete: true` means that your profile will not be ignored.
+... `title`: is a short profile title.
+... `description`: is a few paragraphs that describe the purpose of the profile.
+... `selections`: is a list of rules and variables that consist of the profile. It cannot be an empty list, so for now we will add rule `security_patches_up_to_date`. We will discuss how to find and add other rules later in this track.
+. Save the profile file and go back to the root directory.
++
+----
+cd ../..
+----
++
+. Rebuild the content.
++
+----
+./build_product rhel8
+----
++
+** this command will rebuild contents for all the profiles for the Red Hat Enterprise Linux product including you new “Travel” profile
+** it will build the human-readable HTML guide which can be displayed in a web browser and machine-readable SCAP files that can be consumed by OpenSCAP
+. Check the result HTML guide to look at your new profile.
++
+----
+firefox build/guides/ssg-rhel8-guide-travel.html
+----
++
+** A Firefox Window will open and you will see the guide your Travel profile which contains just a single rule - `security_patches_up_to_date`.
+
+.The header of the HTML Guide generated by OpenSCAP during the build.
+image::2-01-guide.png[HTML Guide]
+
+== Adding rules to the profile
+
+Let’s imagine that one of the requirements of your company policy is that root user cannot log in to the machine via SSH. At this point, we can reveal to you that ComplianceAsCode already contains a rule that implements this requirement. You now only need to add this rule to your “travel” profile.
+
+. Find the relevant rule
```

Comment:
Could you slightly rethink the structure of the following list?

Some of the items are very short (item 3 is just one line, but that's probably a mistake). Also I am not sure whether it is beneficial to use bullet lists instead of paragraphs.